epub33/reports/a11y-properties-use.md

```diff
@@ -66,25 +86,41 @@ implementations.
     <thead>
         <tr>
             <th>Role</th>
-            <th>[Publisher]</th>
+            <th>Used By</th>
         </tr>
     </thead>
     <tbody>
         <tr>
         	<td>dcterms:conformsTo</td>
```

Comment:
As a matter of consistency, shouldn't these be a `<th>` and not a `<td>`? It is the case everywhere else (including the other document).

Reply:
Each row is a property, so this seems unnecessary.

If you're referring to the element/attribute/property definitions in the core spec, each row is a new piece of information. That's why the first cell in each row is a th rather than there being a th that covers the whole column.

Reply:
Oh, I see what you mean. The other document isn't correct. I did a global replace to format the rows and looks like I incorrectly input th for the property names. Going to undo that now.